.bazelrc

```diff
@@ -1,5 +1,5 @@
 # Import Aspect bazelrc presets
-import %workspace%/.aspect/bazelrc/bazel7.bazelrc
+try-import %workspace%/.aspect/bazelrc/bazel7.bazelrc
```

Comment:
Why `try-import`?

Reply:
The try-import is for local development. On CI, to support testing on both bazel 6 and 7, ci.yaml deletes this file (which is a symlink):
```
          # Delete all the version specific bazelrc files that are used for local development
          # since the version we're testing against is dynamic. These are just symlinks and the
          # root .bazelrc brings these in with try-imports. In this CI workflows, we explicitly
          # bring in the version specific bazelrc file with --bazelrc when we invoke bazel.
          rm ${GITHUB_WORKSPACE//\\/\/}/.aspect/bazelrc/bazel*.bazelrc
```
and then explicitly includes the right version:
```
--bazelrc=${GITHUB_WORKSPACE//\\/\/}/.github/workflows/bazel${{ matrix.bazel-version.major }}.bazelrc \
```